plugins/woocommerce-blocks/docs/internal-developers/templates/README.md

```diff
@@ -40,17 +40,18 @@ The BlockTemplateController.php is primarily responsible for hooking into both W
 
 ### Some things to be aware of
 
--   Individual templates are represented by a placeholder block within the Site Editor until we can 'block-ify' these. This is the long-term goal. Until then users will be able to customize templates by adding blocks above and below the placeholder.
+-   For each template, we have a version represented by a placeholder block and the blockified one, which uses more granular blocks. That was done to keep backwards compatibility for extensions in existing stores.
```

Comment:
"and a blockified one"

Reply:
Good catch, changed it in fd43ee297987928218dcff860988888f901220dd.